### ISSUE TITLE
Fightcade Download not working

### ISSUE THREAD

**Issue by ZelphirKaltstahl** [poliva/fightcade-links#1]
Didn't know where else to contact you or whom else I should contact about this:
The Download link for the Linux version of Fightcade did not work for some time. Several browsers I tries said stuff like "too many redirects", "this site doesn't redirect properly" and when I tried it in google translated version, it told me the file http://www.fightcade.com/download/fightcade-linux-v040.tar.gz is not there. Maybe you have them on another devide and that was offline?

**Comment by poliva** [comment]
weird, it's working fine from here.

**Comment by ZelphirKaltstahl** [comment]
Yes it worked for me as well a few minutes ago. I first wanted to write it doesn't work at all, but then I tested again and had to change my writing. So it seems to be a temporary issue.
